client/me/notification-settings/settings-form/settings.jsx

```diff
@@ -1,8 +1,7 @@
 import { find, get } from 'lodash';
```

Comment:
Care to do some more Lodash refactor?

Reply:
Done via https://github.com/Automattic/wp-calypso/pull/63507/commits/5393d8c626bec81bccd8419f41ffc4328dbde9e2